_includes/pricing/faq.html

```diff
@@ -0,0 +1,28 @@
+<div class="k-faq" style="margin-top: 100px">
+  <h4 data-i18n="faq:header.slogan"></h4>
```

Comment:
This should rather be an h2 so it matches the styles of the headers in the home page